### ISSUE TITLE
Static files should be fetched from official repo

### ISSUE THREAD

**Issue by jum-s** [allmende/ofn-docker#1]
For some reason, static files from `public` folder were imported manually into Docker compose.
It should be fetched from [official commited folder](https://github.com/openfoodfoundation/openfoodnetwork/tree/master/public) in order to keep track of any future commits.

**Comment by almereyda** [comment]
Yes, that happened as we ran into a race condition between the build time filesystem layers and the only later added volumes, similar to https://github.com/docker/docker/issues/14080
